<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindWindow</class>
 <widget class="QMainWindow" name="FindWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_find">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_find_Previous">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>查找上一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_find_all">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>50</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>全局查找</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_replace">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_find_Next">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>查找下一个</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_replace_all">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>全部替换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_replace">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>替换当前字符</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
